WooCommerce/Classes/POS/Presentation/CardReaderConnection/UI States/Connection Alerts/PointOfSaleCardPresentPaymentConnectingFailedUpdateAddressView.swift

```diff
@@ -3,20 +3,23 @@ import SwiftUI
 struct PointOfSaleCardPresentPaymentConnectingFailedUpdateAddressView: View {
     @StateObject var viewModel: PointOfSaleCardPresentPaymentConnectingFailedUpdateAddressAlertViewModel
     var body: some View {
-        VStack {
+        VStack(spacing: PointOfSaleReaderConnectionModalLayout.verticalSpacing) {
             Text(viewModel.title)
 
             viewModel.image
 
-            if let primaryButtonViewModel = viewModel.primaryButtonViewModel {
-                Button(primaryButtonViewModel.title,
-                       action: primaryButtonViewModel.actionHandler)
-            }
+            VStack(spacing: PointOfSaleReaderConnectionModalLayout.buttonSpacing) {
+                if let primaryButtonViewModel = viewModel.primaryButtonViewModel {
+                    Button(primaryButtonViewModel.title,
+                           action: primaryButtonViewModel.actionHandler)
+                }
 
-            Button(viewModel.cancelButtonViewModel.title,
-                   action: viewModel.cancelButtonViewModel.actionHandler)
-            .buttonStyle(SecondaryButtonStyle())
+                Button(viewModel.cancelButtonViewModel.title,
+                       action: viewModel.cancelButtonViewModel.actionHandler)
+                .buttonStyle(SecondaryButtonStyle())
+            }
         }
+        .multilineTextAlignment(.center)
         .sheet(isPresented: $viewModel.shouldShowSettingsWebView) {
             WCSettingsWebView(adminUrl: viewModel.settingsAdminUrl,
```

Comment:
(Could be unrelated to this PR)

When I tap "Enter Address" and web view is opened. "Done button doesn't" close the modal, it only calls:

```
func settingsWebViewWasDismissed() {
        retrySearchAction()
    }
 ```